tox.ini

```
@@ -69,7 +69,9 @@ description = Run integration tests
 deps =
     git+https://github.com/juju/python-libjuju.git
     pytest
-    git+https://github.com/charmed-kubernetes/pytest-operator.git
+    # This version constraint is required until the issue below is fixed:
+    # https://github.com/charmed-kubernetes/pytest-operator/issues/72
+    pytest-operator<0.17.0
```

@marceloneppel 

would you be willing to try

```suggestion
    git+https://github.com/charmed-kubernetes/pytest-operator.git@teardown-reliability
```

---

Thanks for the update, @addyess! I saw that the fix was released, so I kept the published version.